build.fsx

```diff
@@ -349,7 +349,7 @@ Target.create "UnskipAndRevertAssemblyInfo" (fun _ ->
 )
 
 Target.create "BuildSolution_" (fun _ ->
-    MsBuild.runWithDefaults "Build" ["./src/Legacy-FAKE.sln"; "./src/Legacy-FAKE.Deploy.Web.sln"]
+    MSBuild.runWithDefaults "Build" ["./src/Legacy-FAKE.sln"; "./src/Legacy-FAKE.Deploy.Web.sln"]
```

Comment:
To fix the build-script you need to nest this change into a `#if BOOTSTRAP` block

Reply:
The `@if BOOTSTRAP` trick didn't work when building on my laptop as it expected `MsBuild` instead of `MSBuild`.
However Travis and AppVeyor builds both expected `MSBuild` as expected, the cause is probably some stale artifacts on my laptop.

Reply:
Correct. Artefacts will be added to your .nuget folder (in your users directory). Maybe our build script needs to clean them...